lib/siftsciex/verification/payload/check.ex

```diff
@@ -0,0 +1,26 @@
+defmodule Siftsciex.Verification.Payload.Check do
+  @moduledoc """
+  Internal representation and "constructor" for a Sift Science Verficiation Check Request Payload
+  """
+
+  alias Siftsciex.Verification.Payload
+
+  defstruct "$user_id": :empty,
+            "$code": :empty
+  @type t :: %__MODULE__{"$user_id": Payload.payload_string,
+                         "$code": Payload.payload_string}
+
+  @type attributes :: :user_id
+                      | :code
+  @type data :: %{required(attributes) => String.t}
+
+  @spec new(data) :: __MODULE__.t
+  def new(data) do
+    normalized = Enum.map(data, &normalize/1)
+
+    struct(%__MODULE__{}, normalized)
+  end
+
+  def normalize({:user_id, value}), do: {String.to_atom("$user_id"), value}
+  def normalize({:code, value}), do: {String.to_atom("$code"), value}
```

Comment:
You can use the following syntax to create an atom: `:"$user_id"`.
That would make this code a little more readable since it reads like compile-time whereas `String.to_atom/1` reads more like runtime.

Reply:
I was trying to keep it consistent with the rest of the package, but sure I can update it to that if you think it's more readable.

Reply:
I can't get comfortable using `String.to_atom/1` "consistently" 😅 
I guess it's ok to keep consistency.